powerauth-webflow-client/pom.xml

```diff
@@ -148,6 +148,15 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
The client does not require connection to Postgres as far as I know.

Reply:
Good catch, I have change that to H2.